controllers/auth.js

```diff
@@ -32,12 +42,50 @@ const register = async(req, res) => {
     })
 };
 
+// Зміна статусу email-у на "верифікований" та очищення верифікаційного коду в БД після підтвердження пошти в отриманому листі
+const verifyEmail = async(req, res) => {
+    const { verificationToken } = req.params;
+    const user = await User.findOne({ verificationToken }); // перевіряємо чи є в БД користувач з таким кодом
+    if(!user) {
+        throw HttpError(401, 'Email is not found')
+    }
+    await User.findByIdAndUpdate(user._id, { verify: true, verificationToken: "" }); // якщо є такий користувач, то вносимо зміни до БД
+
+    res.json({
+        message: "Congratulations! The email has been successfully verified."
+    })
+};
+
+const resendVerifyEmail = async(req, res) => {
+    const { email } = req.body;
+    const user = await User.findOne({ email });
+    if(!user){
+        throw HttpError(401, 'Email is not found');
+    }
+    if(user.verify) { //якщо користувач вже підтвердив email (user.verify = true)
+        throw HttpError(401, 'The email is already verified')   // не спрацює, бо спочатку Postman видасть "Email already in use"
+    }
+    const verifyEmail = {
+        to: email,
```

Comment:
you have already used this piece of code. Reuse it

Reply:
Done! In 'helpers' directory created the file 'createVerificationEmail.js' with function that was reused in 'register' and 'resendVerifyEmail' functions.